src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        String[] names = {
+                "John",
+                "Andrew",
+                "Kate"};
+
+        String[] data = {
+                "26.04.2019 John 4 50",
+                "05.04.2019 Andrew 3 200",
+                "10.04.2019 John 7 100",
+                "22.04.2019 Kate 9 100",
+                "25.06.2019 John 11 50",
+                "26.04.2019 Andrew 3 150",
+                "13.02.2019 John 7 100",
+                "15.04.2019 Kate 9 100"};
+
+        String dateFrom = "15.04.2019";
+        String dateTo = "31.04.2019";
+
+        String raport = SalaryInfo.getSalaryInfo(names, data, dateFrom, dateTo);

```

Comment:
The variable name `raport` seems to be a typo. It should be `report` to match the standard English spelling and improve code readability.